libs/blocks/caas/caas.js

```diff
@@ -30,6 +32,30 @@ const loadCaas = async (a) => {
   a.insertAdjacentElement('afterend', block);
   a.remove();
 
+  const floodGateColor = getMetadata('floodgatecolor') || '';
+  if (floodGateColor === fgHeaderValue) {
+    state.fetchCardsFromFloodgateTree = true;
+  }
+
+  const { env } = getConfig();
+  const { host } = window.location;
+  let chimeraEndpoint = 'www.adobe.com/chimera-api/collection';
+
+  if (host.includes('stage.adobe') || env?.name === 'local') {
+    chimeraEndpoint = S_CAAS_AIO;
+  } else if (host.includes('.hlx.')) {
+    // If invoking URL is not an Acom URL, then switch to AIO
+    chimeraEndpoint = P_CAAS_AIO;
+  }
+
+  if (host.includes('hlx.page') || env?.name === 'local') {
+    // Set draft=true if URL consisting hlx.page (if published url or hlx.live, then draft is false)
```

Comment:
nit: no need for this comment and line 56 comment

Reply:
Updated, thanks